scheduler/kubernetes/wp-k8s.md

```diff
@@ -5,6 +5,8 @@ keywords: portworx, Wordpress, application stack, kubernetes, Persistent Storage
 sidebar: home_sidebar
 redirect_from:
 meta-description: "This article describes how operate WordPress on Kubernetes. You will learn how to run a high performance MySQL database for WordPress as well as use shared volumes for file uploads."
+redirect_from:
+  - /applications/wp-k8s.html
```

Comment:
is this the only page that is changing url in the whole PR?  Glad to see the redirect in place.  Just need to make sure we do that for all pages.  Including if we delete a page (instead of redirecting), we need to set up a redirect to the best location, even if it is just docs home.